src/batched/KokkosBatched_LU_Decl.hpp

```diff
@@ -16,7 +16,7 @@ namespace KokkosBatched {
       KOKKOS_INLINE_FUNCTION
       static int
       invoke(const AViewType &A,
-             const typename MagnitudeScalarType<typename AViewType::non_const_value_type>::type tiny = 0);
+             const typename Kokkos::Details::ArithTraits<typename AViewType::non_const_value_type>::mag_type tiny = 0);
     };       
```

Comment:
This is wrong. In the batched routines, input value_type can be two -  double or SIMD<double> type. When SIMD<double> type is used, your modification returns SIMD<mag_type<double> > but the previous routine MagnitudeScalarType returns double type.

Reply:
Should we keep ```MagnitudeScalarType```?

Reply:
You have two options: keep MagnitudeScalarType or using the ValueScalarType
```
typename Kokkos::Details::ArithTraits<typename ValueScalarType<typename AViewType::non_const_value_type>::type>::mag_type
```

Reply:
I think we should go back to MagnitudeScalarType